modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -92,6 +99,19 @@ export default {
     };
   },
   computed: {
+    label () {
+      const { label, publishedLabel } = this.field;
```

Comment:
OK.

Reply:
@boutell I didn't have to use `draftLabel` in the end, I think we can manage with the  default `label` and `publishedLabel` one